pom.xml

```diff
@@ -1548,6 +1548,7 @@
         <findbugs.skip>true</findbugs.skip>
         <skipITs>true</skipITs>
         <skipTests>true</skipTests>
+        <unitTestMemSize>-Xmx1G</unitTestMemSize>
```

Comment:
I think this was intended to be placed in the mail properties section, not the one inside this profile. A simple mistake. :)